CMakeLists.txt

```diff
@@ -5,16 +5,17 @@ if (POLICY CMP0140)
 	cmake_policy(SET CMP0140 NEW)
 endif ()
```

Comment:
Simplify these by setting `cmake_minimum_required (min...max)`. It haa tge same effect as `cmake_policy(SET NEW)`